dorm/dorm/Presenter/RoomManager/RoomManagerView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  RoomManagerView.swift
+//  dorm
+//
+//  Created by JongHo Park on 2022/07/19.
+//
+
+import UIKit
+
+final class RoomManagerView: UIView {
+
+    // MARK: - class Life cycle
+    init() {
+        super.init(frame: .zero)
+        commomInit()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commomInit()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Don`t need coder initializer")
+    }
+
+    /// UI 요소를 추가하는 함수
+    private func commomInit() {
```

Comment:
```suggestion
    private func setupViews() {
```

메소드 일므이 조금 더명확해지면 좋을 듯해용

Reply:
cb5d50b7b0f1e98c25ece0ca5e3251fc115a823c 
에서 적용완료했습니다! 너무 샘플 코드를 복붙했네요...